### DISCUSSION TITLE
Rlsbb search plugin

### DISCUSSION THREAD

**Discussion by stev412** [LivingWithHippos/unchained-android#213] [Feedback]
The rlsbb search plug-in used to work perfectly but now I don't get any search results, is there something I can try to fix it?

**Comment by LivingWithHippos** [comment]
I tried a search directly on their websites and it does not work so I think you ll have to wait for them to fix it. If they decide to use cloudflare (I saw some of it on the rest of the website) there is nothing I can do. Can you try and use the search on their website?

**Reply by stev412** [reply]
Search seems to work on from the "main" page only for me on their website.

I start here https://rlsbb.to/

And type in a search term which takes me to this address with results ok. 

https://search.rlsbb.to/?s=something

If I then try to search again , from that new result page search box , I get a blank page error , with a different url

https://search.search.rlsbb.to/?s=something

Not sure if that makes any sense

**Reply by LivingWithHippos** [reply]
Yeah of course. I checked a little bit more and there's a mix between them adding link protection (can't parse them) and cloudflare. I think I may end up removing the plugin if it's not possible to get meaningful results but I need to check better

**Comment by LivingWithHippos** [comment]
fixed with 67a70a94
